### PR TITLE
Run `yarn dedupe`

### PR DESCRIPTION
**Pull request by basil**
This PR deduplicates dependencies with overlapping ranges. From [the documentation](https://yarnpkg.com/cli/dedupe):

> Duplicates are defined as descriptors with overlapping ranges being resolved and locked to different locators. They are a natural consequence of Yarn's deterministic installs, but they can sometimes pile up and unnecessarily increase the size of your project.

[…]

> Note: Even though it never produces a wrong dependency tree, this command should be used with caution, as it modifies the dependency tree, which can sometimes cause problems when packages don't strictly follow semver recommendations. Because of this, it is recommended to also review the changes manually.

```
$ yarn dedupe
➤ YN0000: ┌ Deduplication step
➤ YN0000: │ @babel/compat-data@npm:^7.13.11 can be deduped from @babel/compat-data@npm:7.18.6 to @babel/compat-data@npm:7.18.8
➤ YN0000: │ @babel/helper-annotate-as-pure@npm:^7.10.1 can be deduped from @babel/helper-annotate-as-pure@npm:7.10.1 to @babel/helper-annotate-as-pure@npm:7.18.6
➤ YN0000: │ @babel/helper-create-regexp-features-plugin@npm:^7.10.1 can be deduped from @babel/helper-create-regexp-features-plugin@npm:7.10.1 to @babel/helper-create-regexp-features-plugin@npm:7.18.6
➤ YN0000: │ @babel/helper-environment-visitor@npm:^7.18.6 can be deduped from @babel/helper-environment-visitor@npm:7.18.6 to @babel/helper-environment-visitor@npm:7.18.9
➤ YN0000: │ @babel/helper-function-name@npm:^7.18.6 can be deduped from @babel/helper-function-name@npm:7.18.6 to @babel/helper-function-name@npm:7.18.9
➤ YN0000: │ @babel/helper-member-expression-to-functions@npm:^7.18.6 can be deduped from @babel/helper-member-expression-to-functions@npm:7.18.6 to @babel/helper-member-expression-to-functions@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.0.0 can be deduped from @babel/helper-plugin-utils@npm:7.10.1 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.10.1 can be deduped from @babel/helper-plugin-utils@npm:7.10.1 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.8.0 can be deduped from @babel/helper-plugin-utils@npm:7.10.1 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.10.4 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.12.13 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.13.0 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.14.5 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.18.6 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-plugin-utils@npm:^7.8.3 can be deduped from @babel/helper-plugin-utils@npm:7.18.6 to @babel/helper-plugin-utils@npm:7.18.9
➤ YN0000: │ @babel/helper-replace-supers@npm:^7.18.6 can be deduped from @babel/helper-replace-supers@npm:7.18.6 to @babel/helper-replace-supers@npm:7.18.9
➤ YN0000: │ @babel/plugin-proposal-unicode-property-regex@npm:^7.4.4 can be deduped from @babel/plugin-proposal-unicode-property-regex@npm:7.10.1 to @babel/plugin-proposal-unicode-property-regex@npm:7.18.6
➤ YN0000: │ @babel/plugin-transform-dotall-regex@npm:^7.4.4 can be deduped from @babel/plugin-transform-dotall-regex@npm:7.10.1 to @babel/plugin-transform-dotall-regex@npm:7.18.6
➤ YN0000: │ domelementtype@npm:^2.0.1 can be deduped from domelementtype@npm:2.0.1 to domelementtype@npm:2.3.0
➤ YN0000: │ estraverse@npm:^5.1.0 can be deduped from estraverse@npm:5.1.0 to estraverse@npm:5.3.0
➤ YN0000: │ merge2@npm:^1.3.0 can be deduped from merge2@npm:1.3.0 to merge2@npm:1.4.1
➤ YN0000: │ path-parse@npm:^1.0.6 can be deduped from path-parse@npm:1.0.6 to path-parse@npm:1.0.7
➤ YN0000: │ picomatch@npm:^2.0.4 can be deduped from picomatch@npm:2.2.2 to picomatch@npm:2.3.1
➤ YN0000: │ picomatch@npm:^2.2.1 can be deduped from picomatch@npm:2.2.2 to picomatch@npm:2.3.1
➤ YN0000: │ regenerate@npm:^1.4.0 can be deduped from regenerate@npm:1.4.0 to regenerate@npm:1.4.2
➤ YN0000: │ resolve@npm:^1.10.0 can be deduped from resolve@npm:1.17.0 to resolve@npm:1.22.1
➤ YN0000: │ 26 packages can be deduped using the highest strategy
➤ YN0000: └ Completed
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0019: │ resolve-patch-3775f38cdc-6fd799f282.zip appears to be unused - removing
➤ YN0019: │ unicode-canonical-property-names-ecmascript-npm-1.0.4-8c5eeb73e7-cc1973b18d.zip appears to be unused - removing
➤ YN0019: │ unicode-match-property-ecmascript-npm-1.0.4-4729801dd7-08e269fac7.zip appears to be unused - removing
➤ YN0019: │ unicode-match-property-value-ecmascript-npm-1.2.0-d6b5d66edf-2e663cfec8.zip appears to be unused - removing
➤ YN0019: │ unicode-property-aliases-ecmascript-npm-1.1.0-2d3021f23b-1a96dc462d.zip appears to be unused - removing
➤ YN0000: └ Completed in 0s 303ms
➤ YN0000: ┌ Link step
➤ YN0000: │ ESM support for PnP uses the experimental loader API and is therefore experimental
➤ YN0000: └ Completed
➤ YN0000: Done with warnings in 0s 624ms
```

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6913"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

**Comment by basil** [comment]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Comment by basil** [comment]
De minimis

**Pull request merged**
